tools/build/Library/ReleaseCreator.php

```diff
@@ -370,7 +370,7 @@ protected function setConfigDefinesConstants()
      */
     protected function getCurrentVersion()
     {
-        $kernelFile = $this->projectPath.'/app/AppKernel.php';
+        $kernelFile = $this->projectPath.'/src/Core/Version.php';
         $matches = [];
```

Comment:
Maybe we can require the file and load the constant value instead of parsing the file ?

Reply:
Well, I would not make too many changes in this PR as I want to fix an issue for people relying on the nightly.

Plus, we try to generate a release here from the source file **only**. Composer is not supposed to be run yet, so the autoloader cannot work to load other files such as the Version file. That why this "dirty" solution was chosen in the first place.

Reply:
There is no need for composer here

```
require_once $this->projectPath.'/src/Core/Version.php';
return \PrestaShop\PrestaShop\Core\Version::VERSION;
```

Reply:
Very well, here you go